cmd/kogito/command/shared/install_services.go

```diff
@@ -15,11 +15,30 @@
 package shared
 
 import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"reflect"
+
 	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/message"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	kogitocli "github.com/kiegroup/kogito-cloud-operator/pkg/client"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+var defaultReplicas = int32(1)
+var defaultServiceStatus = v1alpha1.KogitoServiceStatus{ConditionsMeta: v1alpha1.ConditionsMeta{Conditions: []v1alpha1.Condition{}}}
+var defaultServiceSpec = v1alpha1.KogitoServiceSpec{Replicas: &defaultReplicas}
+
+type serviceInfoMessages struct {
+	errCreating                  string
+	installed                    string
+	checkStatus                  string
+	notInstalledNoKogitoOperator string
+	serviceExists                string
```

Comment:
not used ?

Reply:
It's indeed: https://github.com/kiegroup/kogito-cloud-operator/pull/318/files#diff-e787bd9ddec738a5228b3b1931d0edb1R109-R113

Basically all supporting services installations use it.